src/commands/hardis/project/deploy/sources/dx.ts

```diff
@@ -387,13 +387,15 @@ If you need to increase the deployment waiting time (force:source:deploy --wait
       forceSourceDeployOptions,
     );
 
+    const deployExecuted = !this.checkOnly && deployXmlCount > 0 ? true : false;
+
     // Set ListViews to scope Mine if defined in .sfdx-hardis.yml
-    if (this.configInfo.listViewsToSetToMine && this.checkOnly === false) {
+    if (this.configInfo.listViewsToSetToMine && deployExecuted) {
       await restoreListViewMine(this.configInfo.listViewsToSetToMine, this.org.getConnection(), { debug: this.debugMode });
     }
 
     // Send notification of deployment success
-    if (!this.checkOnly) {
+    if (deployExecuted) {
```

Comment:
@legetz why removing the test on checkonly ? it can be equal to true with deployExecuted > 0 if we simulate a deployment during a PR check ? :)

Reply:
Same logic here than above